dex/asset.go

```diff
@@ -15,10 +15,65 @@ func (err Error) Error() string { return string(err) }
 
 const (
 	UnsupportedScriptError = Error("unsupported script type")
-	LockTimeTaker          = 24 * time.Hour
-	LockTimeMaker          = 48 * time.Hour
+	defaultLockTimeTaker   = 24 * time.Hour
+	defaultlockTimeMaker   = 48 * time.Hour
 )
 
+var (
+	// These string variables are defined to enable setting custom locktime values
+	// that may be used instead of `DefaultLockTimeTaker` and `DefaultLockTimeMaker`
+	// IF running on a test network (testnet or regtest) AND both values are set to
+	// valid duration strings.
+	// Values for both variables may be set at build time using linker flags e.g:
+	// go build -ldflags "-X 'decred.org/dcrdex/dex.testLockTimeTaker=10m' \
+	// -X 'decred.org/dcrdex/dex.testLockTimeMaker=20m'"
+	// Same values should be set when building server and client binaries.
+	testLockTimeTaker string
+	testLockTimeMaker string
+
+	testLockTime struct {
+		taker time.Duration
+		maker time.Duration
+	}
```

Comment:
I've made both default and test locktime values unexported, and only exported `LockTimeTaker(network)`and `LockTimeMaker(network)` functions to force accessors to provide information about the running network.

One other thought was to export `DefaultLockTime{M,T}aker` and `TestLockTime{M,T}aker` and allow accessors perform network checks before accessing either set of variables but that may make it easy for the wrong set of locktime vars to be used. I think it also prevents avoidable repetition of the `if net == Mainnet` code checks from different usage sites.